content/events/2024-07-27-Urbit-New-England-Xthirdmeetup.md

```diff
@@ -0,0 +1,8 @@
++++ title = "Urbit Meetup in Somerville, MA - Aeronaut Brewing" starts = "2024-07-27T14:00:00" ends = "2024-07-27T17:00:00" timezone = "America/New_York" location = "Aeronaut Brewing - 14 Tyler St, Somerville, Massachusetts 02143" image = "https://media.urbit.org/site/ecosystem/communities/new-england.jpg" registration_url = "https://www.meetup.com/urbit-new-england-group/events/302151131/" description = "The third meetup for Urbiters in New England" dark = true darken_image = "true"
+
++++
```

Comment:
These attributes need to be newline delimited. When using existing events as reference, view them in raw mode first. The markdown preview shows the rendered output, not the raw source content

Reply:
fixed. I'll remember this for next time. Thanks.